cli/src/command.rs

```diff
@@ -328,14 +328,12 @@ where
 	};
 
 	runner.run_node_until_exit(move |config| async move {
-		let hwbench = if !cli.run.no_hardware_benchmarks {
-			config.database.path().map(|database_path| {
+		let hwbench = (!cli.run.no_hardware_benchmarks)
+			.then_some(config.database.path().map(|database_path| {
 				let _ = std::fs::create_dir_all(&database_path);
 				sc_sysinfo::gather_hwbench(Some(database_path))
```

Comment:
shouldn't be there 3 args?
https://github.com/paritytech/substrate/pull/12620/files#diff-933dca0c14add2a08eaca93fe746ada8e72f547d2861ca2079f191908984f06fR594

Reply:
Yea but i am un-doing that here now https://github.com/paritytech/substrate/pull/13198  
I will ping again when the comp check fails